src/main/java/org/antipathy/scalafmtmvn/FormatMojo.java

```diff
@@ -5,31 +5,43 @@
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+
+import java.util.List;
 
 /**
  * Get the location of the config file and pass to Formatter
  */
 @Mojo(name = "format")
 public class FormatMojo extends AbstractMojo {
 
-    @Parameter(property="format.configLocation",defaultValue="")
+    @Parameter(property = "format.configLocation")
     private String configLocation;
-    @Parameter(property="format.parameters",defaultValue="")
+    @Parameter(property = "format.parameters")
     private String parameters;
+    @Parameter(defaultValue = "${project}", required = true, readonly = true)
```

Comment:
Actually this param is not visible by user, it's a Maven property, but I did update the version and fixed some typos in the README.ms.